.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -11,8 +11,8 @@ body:
 
         This is the repository for the command-line program `quarto`:
 
-        - If you're reporting an issue with the VS Code extension, please visit https://github.com/quarto-dev/quarto
-        - If you're reporting an issue inside RStudio, please visit https://github.com/rstudio/rstudio
+        - If you're reporting an issue with the **Visual Studio Code extension**, please visit https://github.com/quarto-dev/quarto
```

Comment:
If we're editing this, we should try and find wording for the extension line that includes both Visual Studio Code and Positron (users might not realize that the Positron extension and the VS code extension are the same thing)